src/pages/guides/tutorials/grids-addon.md

```diff
@@ -0,0 +1,1321 @@
+```
+
+The Reference also shows an interesting [`blendMode`](/references/scriptruntime/editor/classes/GroupNode.md#blendmode): setting it to [`multiply`](/references/scriptruntime/editor/enums/BlendModeValue/#multiply) will produce a visually nicer overlay effect ([opacity](/references/scriptruntime/editor/classes/GroupNode.md#opacity) can be an alternative).
+
+```js
+// ...
+rowsGroup.blendMode = Constants.BlendModeValue.multiply;
+rowsGroup.locked = true;
+```
+
+At present, the Reference includes only a few enumerations, such as `BlendModeValue`. As the Document API expands, more enumerations will be added. They provide sets of named constants, making the code more readable by replacing direct numeric values with descriptive names.[^4]
+
+It would be preferable if a single group contained Rows and Columns; we must edit both `addRows()` and `addColumns()` first to return their group in order to reference them in the parent one.
+
+```js
+const addRows = (rowsNumber, gutter, color) => {
+	// ...
+	rowsGroup.locked = true;
+	return rowsGroup;  // 👈 returning the group
+}
+
+const addColumns = (columNumber, gutter, color) => {
+	// ...
+	columnsGroup.locked = true;
+	return columnsGroup;// 👈 
+}
+```
+
+They can be referenced in `addGrid()`, appending them as children of this new group.
+
+```js
+addGrid({ columns, rows, gutter, columnColor, rowColor }) {
+	// ...
+
+	const rowGroup = addRows(rows, gutter, rowColor);
+	const columnGroup = addColumns(columns, gutter, columnColor);
+
+	// create a parent group
+	const gridGroup = editor.createGroup();
+	page.artboards.first.children.append(gridGroup);
+	gridGroup.children.append(rowGroup, columnGroup); // filling with Rows and Columns
+	gridGroup.locked = true;
+}
+```
+
+### Deleting Grids
+
+From the users' perspective, it would speed up the testing process of new grid designs if they could clear the current one and experiment with different combinations—that is, a Delete button. You can both add and remove elements to a container object's `children` list: most Node elements have a `removeFromParent()` method, which detaches them from their parent container.
+
+```js
+gridGroup.removeFromParent(); // voilà
+```
+
+Curb your enthusiasm: if you think about it, there might be an issue lurking here. The process of removing a group is performed at a later time, with respect to its creation: how can we be sure to target the correct container? Groups don't have names or IDs, at least for the time being. Luckily, the `code.js` file provides **an environment that persists in between iFrame calls**: in other words, you can store the group in a variable and retrieve it later.
+
+```js
+var gridRef = null; // 👈 Grids group reference
+
+function start() {
+  runtime.exposeApi({
+    addGrid({ columns, rows, gutter, columnColor, rowColor }) {
+      // ...
+      const gridGroup = editor.createGroup();
+      // ...
+      gridRef = gridGroup; // 👈 storing the group for later
+    },
+    deleteGrid() {
```

Comment:
Since we can return results back to the iFrame, maybe instead of just logging to the console, we also return a failure code?